extensions/sql-database-projects/package.json

```diff
@@ -382,18 +401,22 @@
     }
   },
   "dependencies": {
-    "@types/xml-formatter": "^1.1.0",
     "@microsoft/ads-extension-telemetry": "^1.1.5",
+    "@types/which": "^2.0.1",
+    "@types/xml-formatter": "^1.1.0",
```

Comment:
@Benjin `@types` packages should never go in dependencies, they don't have any runtime components.